code/mob/living/carbon/human/procs/Life.dm

```diff
@@ -1835,11 +1795,8 @@
 			src.take_eye_damage(-1, 1)
 			src.blinded = 1
 
-		// drsingh :wtc: why was there a runtime error about comparing "" to 50 here? varedit or something?
-		// welp thisll fix it
-		parent.setLastTask("status_updates disability checks", src)
-		src.stuttering = isnum(src.stuttering) ? min(src.stuttering, 50) : 0
```

Comment:
stuttering was "sort-of" clamped in here and `clamp_values()`, I made the `clamp_values()` clamp work better and removed this.